src/main/scala/scalismo/io/StatisticalModelIO.scala

```diff
@@ -15,7 +15,9 @@
  */
 package scalismo.io
 
-import java.io._
+import scalismo.common.interpolation.NearestNeighborInterpolator3D
+
+import java.io.*
```

Comment:
Can we make scalafmt sorting the includes?

Reply:
I will look into it and make a separate PR with a new scalafmt. Once we got rid of scala 2.13 we have a lot more freedom to clean up things and enforce new formatting rules.